README.md

```diff
@@ -467,34 +470,46 @@ notification intent that the Klaviyo SDK requires in order to track opens when y
 
 ## In App Forms
 
+In-app forms are messages displayed to mobile app users while they are actively using your app. 
+You can create new in-app forms in a drag-and-drop editor in the Sign-Up Forms tab in Klaviyo.
+
+[//]: # (TODO -- add help doc link)
+
 ### Prerequisites
-- Using version 3.2.0 or higher
-- You need to have Klaviyo `analytics` imported and the company ID must be set
+- Version 3.2.0 or higher
+- Klaviyo `analytics` and `forms` packages
 
 ### Setup
-`registerForInAppForms` can be chained with your initialization code to allow for form display.
+Once you've created an in-app form in your Klaviyo account, to display it add the following code to your application. 
```

Comment:
You don't technically need to create an in-app form in your account before adding this code to your app. Should we just say "To display in-app forms, add the following code..."